lib/ama_layout/navigation.yml

```diff
@@ -19,13 +19,17 @@ member:
     link: "<%= Rails.configuration.driveredonline_site %>"
     icon: "fa-car"
   - text: "My Reward Dollars"
-    link: "<%= Rails.configuration.youraccount_site %>/reward_dollars"
+    link: "<%= Rails.configuration.youraccount_site %>/rewards"
     icon: "fa-usd"
-  - text: "Reload Esso Gift Cards"
-    link: "<%= Rails.configuration.membership_site %>/reward_cards"
-    icon: "fa-arrow-up"
+    sub_nav:
+      - text: "Rewards Overview"
+        link: "<%= Rails.configuration.youraccount_site %>/rewards"
+      - text: "Reload Esso Gift Cards"
+        link: "<%= Rails.configuration.membership_site %>/reward_cards"
+      - text: "Transaction History"
+        link: "<%= Rails.configuration.youraccount_site %>/reward_dollars"
   - text: "My Account Settings"
-    link: "#"
+    link: "<%= Rails.configuration.youraccount_site %>/subscriptions"
```

Comment:
This was # because clicking the top level menu is only supposed to expand the menu now - should be left at #